app/serializable/serializable_resource.rb

```diff
@@ -14,7 +14,11 @@ class SerializableResource < SerializableJSONAPIResource
              :isPeerReviewed,
              :isTitleCustom,
              :publisherName,
-             :titleId
+             :titleId,
+             :isPeerReviewed,
+             :publisherName,
```

Comment:
several of these are dupes - `publisherName` and `description` on first glance

Reply:
Oops, fixed.